ffi/src/initialization.rs

```diff
@@ -46,6 +46,7 @@ pub const CPU_PROVIDER_ID: u32 = u32::MAX;
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 #[allow(clippy::upper_case_acronyms)]
 pub enum DeviceClass {
+    UNSPECIFIED,
```

Comment:
Now it is complaining that it's a useless code. And it is hard to disagree with it :)

As I understood actually GRPC API will be handled on the go-spacemesh side.
So probably we don't even need UNSPECIFIED value here, but enumerate options starting from one:
```
pub enum DeviceClass {
  CPU = 1,
  GPU = 2,
}
```

And then we'll need to handle the case with UNSPECIFIED value only on the API handler level, not in the actual business logic.

Wdyt?

Reply:
Sounds good! 👍